app/globals.css

```
@@ -8,10 +8,11 @@
 html {
   scroll-behavior: smooth;
 }
-@import "react-image-crop/dist/ReactCrop.css";
+@import 'react-image-crop/dist/ReactCrop.css';
```

Stylelint (17.11.0)

[error] Invalid position for `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)